microsoft-edge/web-platform/ie-to-microsoft-edge-redirection.md

```diff
@@ -5,28 +5,15 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
-ms.date: 11/13/2020
+ms.date: 2/8/2022
 ---
 # Move users to Microsoft Edge from Internet Explorer
 
-Many modern websites have designs that are incompatible with Internet Explorer.  When an Internet Explorer user visits an incompatible public website, the user may get a message.  The message states that the website is incompatible with the browser.  After the message is displayed, the user is expected to manually switch to a modern browser.
+Many modern websites have designs that are incompatible with Internet Explorer.
 
-To minimize disruptions, starting with version 84, Microsoft Edge supports a new capability that automatically redirects users.  When an Internet Explorer user goes to a website that's incompatible with Internet Explorer, Windows automatically redirects the user to Microsoft Edge.  For the list of websites that are redirected, see [Need Microsoft Edge list](https://edge.microsoft.com/neededge/v1).
+When an Internet Explorer user visits an incompatible public website, the user may be instructed by the website that it is incompatible with Internet Explorer and to switch to a more modern browser.
```

Comment:
```suggestion
When an Internet Explorer user visits an incompatible public website, the user may be instructed by the website that the website is incompatible with Internet Explorer, and that the user must switch to a more modern browser in order to use the website.
```
"that it is" and "to switch" aren't parallel enough, IMO; hard to parse.  "It" is less clear than could be.